process/main.py

```diff
@@ -184,6 +185,12 @@ def parse_args(self, args):
             default="MFILE.DAT",
             help="mfile for post-processing/plotting",
         )
+        parser.add_argument(
+            "-mj",
+            "--mfilejson",
+            action="store_true",
+            help="produce a json containing mfile data",
```

Comment:
Update the help message slightly. Where does it write the JSON file, and with what name?

Reply:
parser.add_argument(
            "-mj",
            "--mfilejson",
            action="store_true",
            help="Produce a filled json from --mfile arg in working dir",
        )
Modified like this